besca/Import/_read.py

```diff
@@ -314,12 +327,13 @@ def check_data_for_citeseq(var_anno):
     """
     if (len(var_anno.columns)) >= 3:
         values = var_anno.iloc[:, 2].unique()
-        if set(values) == set(['Gene Expression']) or \
-            set(values) == set(['Antibody Capture']) or \
-            set(values) == set(['Antibody Capture', 'Gene Expression']):
+        if (
+            set(values) == set(["Gene Expression"])
```

Comment:
The logic is wrong here, we have to remove line 331. See my patch: https://github.com/bedapub/besca/pull/264

Reply:
Yes, these changes were done by my editor. But I'll do it as you suggested